lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -36,6 +36,16 @@ function dosomething_global_init() {
     dosomething_global_redirect_node_edit($node_variables['node'], $user_variables['user_language']);
     return;
   }
+
+  // Verify we're dealing with an admin page that has a path prefix
+  // and if we are, redirect them to that page without a path prefix
+  if ($url_variables['is_admin_page'] && $url_variables['url_path_prefix'] != $languages[DOSOMETHING_GLOBAL_DEFAULT_LANG_CODE]->prefix) {
+    $original_path_without_prefix = str_replace($url_variables['url_path_prefix'], "", $url_variables['path']);
+    $original_path_without_prefix = ltrim($original_path_without_prefix, '/');
+    $redirect_path = url(base_path(), array('absolute' => TRUE, 'language' => 'en-global')) . $original_path_without_prefix;
```

Comment:
I feel like this should be:
`$redirect_path = url($original_path_without_prefix, array('absolute' => TRUE, 'language' => 'en-global'))`